website/pages/docs/auth/jwt.mdx

```diff
@@ -195,6 +195,22 @@ collection of provider configuration steps has been collected to help get starte
 This amount of configuration required for OIDC is relatively small, but it can be tricky to debug
 why things aren't working. Some tips for setting up OIDC:
 
+- If a role parameter (e.g. `bound_claims`) requires a map value, it can't be set individually using
+  the Vault CLI. In these cases the best approach is to write the entire configuration as a single
+  JSON object:
+
+```text
+vault write auth/oidc/role/demo -<<EOF
+{
+  "user_claim": "sub",
+  "bound_audiences": "abc123",
+  "role_type" : "oidc",
+  "policies" : "demo",
+  "ttl" : "1h",
+  "bound_claims" : { "groups": ["myawesomegroup/myawesomesubgroup"] }
```

Comment:
Does this JSON need to be quoted?

Reply:
As this is a map type, I don't think anything needs to be stringified. Which part did you mean?

Reply:
I was curious if this needed to be done:

`"bound_claims" : "{ \"groups\": [\"myawesomegroup/myawesomesubgroup\"] }"`

Reply:
Not needed, nor allowed:  `error converting input { "groups": ["myawesomegroup/myawesomesubgroup"] } for field "bound_claims": '' expected a map, got 'string'`

A framework improvement is to more intelligently handle both cases.